docs/source/config_syntax.md

```diff
@@ -0,0 +1,113 @@
+# MONAI Bundle Configuration
```

Comment:
I think the `OmegaConf` documentation can be a good reference to introduce the regular operations and syntax:
https://omegaconf.readthedocs.io/en/2.1_branch/

Thanks.